opennms-base-assembly/src/main/filtered/etc/snmp-graph.properties.d/activemq-graphs.properties

```diff
@@ -0,0 +1,109 @@
+reports=activemq.total.consumer.count, \
+activemq.total.connection.count, \
+activemq.message.count, \
+activemq.consumer.producer.count, \
+activemq.storage.count, \
+activemq.temp.storage.count
+
+#
+# Consumer Count
+#
+report.activemq.total.consumer.count.name=ActiveMQ Consumer Count
+report.activemq.total.consumer.count.columns=TtlConsumerCnt
+report.activemq.total.consumer.count.type=interfaceSnmp
+report.activemq.total.consumer.count.command=--title="ActiveMQ Consumer Count" \
+ --units-exponent=0 \
+ --vertical-label="Consumer Count" \
+ DEF:val1={rrd1}:TtlConsumerCnt:AVERAGE \
+ AREA:val1#babdb6 \
+ LINE2:val1#888a85:"Consumer Count " \
+ GPRINT:val1:AVERAGE:"Avg \\: %10.2lf" \
+ GPRINT:val1:MIN:"Min \\: %10.2lf" \
+ GPRINT:val1:MAX:"Max \\: %10.2lf\\n"
+
+#
+# Total Connection Count
+#
+report.activemq.total.connection.count.name=ActiveMQ Total Connection Count
+report.activemq.total.connection.count.columns=TtlConCnt
+report.activemq.total.connection.count.type=interfaceSnmp
+report.activemq.total.connection.count.command=--title="ActiveMQ Connection Count" \
+ --units-exponent=0 \
+ --vertical-label="Connection Count" \
+ DEF:val1={rrd1}:TtlConCnt:AVERAGE \
+ AREA:val1#babdb6 \
+ LINE2:val1#888a85:"Connection Count " \
+ GPRINT:val1:AVERAGE:"Avg \\: %10.2lf" \
+ GPRINT:val1:MIN:"Min \\: %10.2lf" \
+ GPRINT:val1:MAX:"Max \\: %10.2lf\\n"
+
+#
+# Message Count
+#
+report.activemq.message.count.name=ActiveMQ Message Count
+report.activemq.message.count.columns=MinMsgSize, AvgMsgSize, MaxMsgSize
+report.activemq.message.count.type=interfaceSnmp
+report.activemq.message.count.command=--title="ActiveMQ Message Count" \
+ --units-exponent=0 \
+ --vertical-label="Message Count" \
+ DEF:val1={rrd1}:MinMsgSize:AVERAGE \
+ DEF:val2={rrd2}:AvgMsgSize:AVERAGE \
+ DEF:val3={rrd3}:MaxMsgSize:AVERAGE \
+ COMMENT:"Message Count\\n" \
+ LINE2:val1#ad7fa8:"Min " \
+ GPRINT:val1:MIN:"%10.2lf\\n" \
+ LINE2:val2#729fcf:"Max " \
+ GPRINT:val2:MAX:"%10.2lf\\n" \
+ LINE2:val3#c17d11:"Avg " \
+ GPRINT:val3:AVERAGE:"%10.2lf\\n"
+
+#
+# Prducer-Consumer Count
+#
+report.activemq.consumer.producer.count.name=ActiveMQ Producer-Consumer Count
+report.activemq.consumer.producer.count.columns=TtlProdCnt,TtlConsumerCnt
+report.activemq.consumer.producer.count.type=interfaceSnmp
+report.activemq.consumer.producer.count.command=--title="ActiveMQ Producer-Consumer Count" \
+ --units-exponent=0 \
+ --vertical-label="Message Count" \
+ DEF:val1={rrd1}:TtlProdCnt:AVERAGE \
+ DEF:val2={rrd2}:TtlConsumerCnt:AVERAGE \
+ COMMENT:"Producer Consumer Count\\n" \
+ LINE2:val1#ad7fa8:"Min " \
+ GPRINT:val1:MIN:"%10.2lf\\n" \
+ LINE2:val2#729fcf:"Max " \
+ GPRINT:val2:MAX:"%10.2lf\\n" \
+
+#
+# Storage
+#
+report.activemq.storage.count.name=ActiveMQ Storage
```

Comment:
This graph doesn't look right, putting the limit and the % used on the same graph throws the scale off.